Document/0x05a-Platform-Overview.md

```diff
@@ -11,21 +11,21 @@ Visit the official [Android developer documentation website](https://developer.a
 
 ## Android Security Architecture
 
-Android is a Linux-based open source platform, initially developed by Google as a mobile operating system (OS) solution. Today, the platform is a foundation for a wide variety of modern technology such as mobile phones and tablets, wearable tech, and other "smart" devices like TVs. Typical Android builds ship with a range of pre-installed ("stock") apps and support installation of third-party apps through the Google Play store and other marketplaces.
+Android is a Linux-based open-source platform initially developed by Google as a mobile operating system (OS) solution. Today, the platform is a foundation for a wide variety of modern technology such as mobile phones and tablets, wearable tech, and other "smart" devices like TVs. Typical Android builds ship with a range of pre-installed ("stock") apps and support installation of third-party apps through Google Play store and other marketplaces.
 
-The software stack of Android is composed of several different layers. Each layer defines certain behavior and offers specific services. Further, these layers interact with OS portions above, below, and other instances paralleling the respective layer’s position within the system architecture.
+The software stack of Android is composed of several different layers. Each layer defines particular behavior and offers specific services. Further, these layers interact with OS portions above, below, and other instances paralleling the respective layer’s position within the system architecture. - ASZ - the phrase is unclear and requires re-phrasing

```

Comment:
>  - ASZ - the phrase is unclear and requires re-phrasing

First, don't put comments like this in the text. Second, the sentence makes more sense in the context of the image shown below it.